src/providers/oracle/qgsoraclesourceselect.h

```diff
@@ -134,7 +134,7 @@ class QgsOracleSourceSelect : public QDialog, private Ui::QgsDbSourceSelectBase
     //!Sets a new regular expression to the model
     void setSearchExpression( const QString& regexp );
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_vector/supported_data.html#id2" ) ); }
```

Comment:
When the help button is clicked, the browser will be opened with the url http://docs.qgis.org/2.14/en/docs/user_manual/working_with_vector/supported_data.html#id2

But you can also download and unpack the docs to e.g. `/usr/share/doc/qgis/user_manual` and configure this as an additional prefix with higher priority to have a local (preferred) doc.